casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala

```diff
@@ -300,6 +300,8 @@ class MultiParentCasperImpl[F[_]
               EquivocationDetector.checkNeglectedEquivocationsWithUpdate(b, s.dag, approvedBlock)
             )
         _      <- EitherT.liftF(Span[F].mark("neglected-equivocation-validated"))
+        _      <- EitherT(Validate.phloPrice(b, casperShardConf.minPhloPrice))
+        _      <- EitherT.liftF(Span[F].mark("phlogiston-price-validated"))
```

Comment:
I am thinking maybe we can exclude validating here because the coop is controlling all the nodes.

Reply:
@zsluedem, thanks for your careful consideration! This is the initial requirement. See the last checkbox in [issue description](https://github.com/rchain/rchain/issues/3314#issue-799496971). Also, as Tomislav answered me in Discord:
> ... you can run your own network.

Reply:
@zsluedem This is a good point. This validation is only to punish validator which accepted lower price deploys. And this scan happen if not configured correctly.

Let's comment out this validation for now.

Reply:
Done